examples/notebooks/rag/rag_1D_query_replicate.ipynb

```diff
@@ -249,33 +249,45 @@
     "\n",
     "### LLM Choices at Replicate\n",
     "\n",
-    "- llama 3.1 : Latest\n",
-    "    - **meta/meta-llama-3.1-405b-instruct** : Meta's flagship 405 billion parameter language model, fine-tuned for chat completions\n",
-    "- Base version of llama-3 from meta\n",
-    "    - [meta/meta-llama-3-8b](https://replicate.com/meta/meta-llama-3-8b) : Base version of Llama 3, an 8 billion parameter language model from Meta.\n",
-    "    - **meta/meta-llama-3-70b** : 70 billion\n",
-    "- Instruct versions of llama-3 from meta, fine tuned for chat completions\n",
-    "    - **meta/meta-llama-3-8b-instruct** : An 8 billion parameter language model from Meta, \n",
-    "    - **meta/meta-llama-3-70b-instruct** : 70 billion\n",
+    "\n",
+    "| Model                               | Publisher | Params | Description                                          |\n",
+    "|-------------------------------------|-----------|--------|------------------------------------------------------|\n",
+    "| ibm-granite/granite-3.0-8b-instruct | IBM       | 8 B    | IBM's newest Granite Model v3.0  (default)           |\n",
+    "| ibm-granite/granite-3.0-2b-instruct | IBM       | 2 B    | IBM's newest Granite Model v3.0                      |\n",
+    "| meta/meta-llama-3.1-405b-instruct   | Meta      | 405 B  | Meta's flagship 405 billion parameter language model |\n",
+    "| meta/meta-llama-3-8b-instruct       | Meta      | 8 B    |                                                      |\n",
```

Comment:
Missing description for the last 2 rows

Reply:
@sujee I think we can add these descriptions for the last 2 rows:

Meta's 8 billion parameter language model
Meta's 70 billion parameter language model